app/routes.py

```diff
@@ -1,2 +1,277 @@
 from flask import Blueprint
+import flask_migrate
+from app import db 
+from app.models.task import Task 
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response 
+from flask import jsonify
+from datetime import date
+import requests
+import os
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+@tasks_bp.route("", methods=["POST", "GET"], strict_slashes=False)
+def handle_task():
+    if request.method == "GET":  
+        query_param_value = request.args.get("sort")
+        if query_param_value == "asc":
+            tasks = Task.query.order_by(Task.title.asc())
+
+        elif query_param_value == "desc":
+            tasks = Task.query.order_by(Task.title.desc())
+
+        else:
+            tasks = Task.query.all()
+        tasks_response = []
+        for task in tasks:
+            tasks_response.append({
+                "id": task.task_id,
+                "title": task.title,
+                "description": task.description,
+                "is_complete": False
+            })
+        return jsonify(tasks_response), 200
+
+    elif request.method == "POST":
+        request_body = request.get_json()
+        if "title" in request_body and "description" in request_body and "completed_at" in request_body:
+            new_task = Task(title = request_body["title"], 
+                        description = request_body["description"], 
+                        completed_at = request_body["completed_at"])
+            db.session.add(new_task)
+            db.session.commit() 
+
+            if new_task.completed_at == None:
+                is_complete = False 
+            else:
+                is_complete = True 
+            
+            return {
+                "task":{
+                    "id": new_task.task_id,
+                    "title": new_task.title,
+                    "description": new_task.description,
+                    "is_complete": is_complete
```

Comment:
again think it's odd that Task has a method for `to_json` but it isn't being used here. This code look the same as that code, I think...?